SpatialGDK/Source/Private/Interop/SpatialTypebindingManager.cpp

```diff
@@ -117,6 +117,20 @@ void USpatialTypebindingManager::CreateTypebindings()
 					Info.HandoverProperties.Add(HandoverInfo);
 				}
 			}
+
+			if (Property->PropertyFlags & CPF_AlwaysInterested)
+			{
+				for (int32 ArrayIdx = 0; ArrayIdx < PropertyIt->ArrayDim; ++ArrayIdx)
```

Comment:
Duplicate code, template it?

Reply:
I removed properties in the FInterestPropertInfo struct due to parts being unused so it's not duplicated any more really.